### PR TITLE
chor(es8): Update docs to reflect the current ES supported versions

### PR DESCRIPTION
**Pull request by andromaqui**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ You can also use newer versions of Desktop and Web Modeler with older Zeebe vers
 | Optimize 3.7.x  | Camunda Platform 7.14.x, 7.15.x, 7.16.x    | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+ |
 | Optimize 3.8.x  | Camunda Platform 7.15.x, 7.16.x, 7.17.x    | OpenJDK 11+               | 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+        |
 | Optimize 3.9.x  | Camunda Platform 7.16.x, 7.17.x, 7.18.x    | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
-| Optimize 3.10.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x    | OpenJDK 11+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
+| Optimize 3.10.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x    | OpenJDK 11+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+, 8.7.0+                                      |
```
Comment (OmranAbazid) [inline docs/reference/supported-environments.md:74]: The in the ticket ATs, it is mentioned:
"Our docs mention that the 3.11 release will be requiring ES 8.7.0+"
This is currently not mentioned anywhere.
If we want to mention it, then we could do something like this:

```suggestion
| Optimize 3.10.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x    | OpenJDK 11+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+, 8.7.0+                                      |
| Optimize 3.11.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x    | OpenJDK 11+               | 8.7.0+  
```

**Review by OmranAbazid — APPROVED** [review]

**Comment by OmranAbazid** [comment]
@andromaqui did the developer experience reviewed this?

**Comment by andromaqui** [comment]
@OmranAbazid no no, they also meed to do it so I cant merge yet!

**Review by christinaausley — APPROVED** [review]
Nice. This is only for the `next` version of docs, yes? If not, will need to make the same adjustments in the current version of documentation.

**Comment by andromaqui** [comment]
@christinaausley yeah its for the next version :)

**Pull request merged**